api/src/main/java/org/openmrs/validator/PersonValidator.java

```diff
@@ -68,41 +68,79 @@ public void validate(Object target, Errors errors) {
 		}
 		
 		Person person = (Person) target;
+		if (person.getVoided()) {
```

Comment:
Is this refactoring or new functionality?

Reply:
No new functionality. I just rearrange this line of code. It was previously in the end. Now I moved it up to make the code easier/more readable:

See line 103: https://github.com/openmrs/openmrs-core/pull/2953/commits/69fba31b11fd3454630bb643e9dcb1685833a079#diff-7014d6c9eb6084f32ffc5c659e8ee524L103

Reply:
We usually do such formatting changes in separate pull requests.

Reply:
I did so because for me it seemed to be a valid refactoring operation. Do you want this Pull Requests to be split up?

Reply:
Because any refactoring has potential to introduce unintended bugs, i find it easier and cleaner to focus on only what the ticket was meant for. You can always create more tickets for further improvements.

Reply:
My understanding was that refactoring to make a method simpler would also mean to reduce over-complex if conditions and make the code easier by layout. Anyways I removed the difference in the if condition and change in order. It should now only contain the new methods.